Monika After Story/game/script-topics.rpy

```diff
@@ -15749,3 +15749,54 @@ label monika_literature_value:
     m 1eku "Honestly, I think if more people valued books and poems a little more, the world would be a much better place."
     m 1hksdlb "That's just my opinion as president of a literature club, though. {w=0.2}I guess most people wouldn't think that deeply about it."
     return
+    
+# Do you like nature?
+default persistent._mas_pm_likes_nature = None
+
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_nature",category=['nature', 'you'],prompt="The outdoors",random=True))
+ 
+label monika_nature:
+    m 2esd "Hey, [player]..."
+    $ _history_list.pop()
+    menu:
+        m "Do you like nature{fast}?"
+        "Yes.":
+            $ persistent._mas_pm_likes_nature = True
+            m 2sub "Really? That's wonderful!"
+            m 3wud "You know, I think nature is something we should cherish."
+            m 3eua "Not only is it beautiful, but it also helps humanity out."
+            m 4esc "Insects pollinate our crops, trees give us wood and shade, pets offer us companionship..."
+            m 3esd "Most of all, autotrophs such as plants and algae produce food and oxygen; without them, most life on Earth wouldn't exist."
+            m 1esa "Because of that, I think it's only fair that we give something back to nature since it does so much for us."
+            m 7hub "So, here's Monika's Green Tip of the Day!"
+            m 7eud "Sometimes, people hesitate to go green because they're worried that it's too expensive."
+            m 4esd "That's partially true. Electric vehicles, smart homes, and solar roofs can all cost a fortune."
+            m 3wuo "However, you can make a difference and {i}save{/i} money just by making a few simple choices each day!"
```

Comment:
![1](https://user-images.githubusercontent.com/45746485/93428892-84952e80-f8c0-11ea-91fa-37d6bf25d3a7.png)

This pose reminds me more of Monika explaining the positive side of something.

Reply:
I like 3wuo here because it conveys the sentence as a type of 'shocking revelation' for the player in that it says "You may think this, but surprisingly you can SAVE money AND go green!". I'm not sure how to put it, but I've seen 3wuo before as the "but actually" type, and I feel like it fits.

Reply:
I understand what you mean, in the prom topic it's used in a very "This is shocking news" kind of way. However, I do not think that this should be a very shocking revelation type sentence. It just doesn't sit right with me, I would never have used a shocked face to explain that someone can save money by going green, and I don't feel Monika would either. It's not like a "the Matrix is real" kind of revelation, it's more of a "this is pretty neat information, right?" type revelation.

Maybe a third opinion is in order here.

Reply:
Did I post twice about this sentence? Woops. Look above for my reply.

Reply:
I think the context of the sentence is important here.

> m 7ruc "Sometimes, people hesitate to go green because they're worried that it's too expensive."
>             m 4esd "That's partially true. Electric vehicles, smart homes, and solar roofs can all cost a fortune."
>             m 3wuo "However, you can make a difference and {i}save{/i} money just by making a few simple choices each day!"

The original claim is that going green can be too expensive because certain products (e.g. electric vehicles) are expensive, and Monika admits to this being partially correct. However, she turns it around—not only can you go green while not spending much money, you can actually SAVE more money than normal by going green. In other words, Monika is making a surprising claim that instead of going green being expensive, it can actually LOWER the player's expenses on utilities. Maybe the original dialogue doesn't describe it as well as I'd hoped, but the surprise comes from an inversion of the original claim that not many people may expect to hear if they're under the impression that the best way to go green is through EVs, solar roofs etc. 

With that said, I can see your point on it being a 'pretty neat info' kind of relevation as well, so I agree that it may be best to wait on a third opinion.

Reply:
I think it's best to just leave the sprites from here and let the devs decide :P They have the most experience in making Monika feel consistent after all. We did good pre-cut work here though.

Reply:
We did indeed. Good work. o7